src/main/java/com/ebivariation/contigalias/api/ClientAPI.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ebivariation.contigalias.api;
+
+import com.ebivariation.contigalias.entities.AssemblyEntity;
+import com.ebivariation.contigalias.service.AssemblyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RequestMapping("contig-alias/api")
+@RestController
+public class ClientAPI {
+
+    private final AssemblyService service;
+
+    @Autowired
+    public ClientAPI(AssemblyService service) {
+        this.service = service;
+    }
+
+    @GetMapping(value = "{accession}")
```

Comment:
I have changed the endpoint to `/conting-alias/assemblies/{accession}`. Should I also add endpoints for features that haven't been implemented yet?

Reply:
it depends on your workflow style. If you add unimplemented endpoints, it helps to get the big picture while testing it, and you should `throw new UnsupportedOperationException();` for anything that is expected not to work.

Reply:
I have added one more endpoint that I will soon implement. I think it will be easier for me too add end points once the data is in a database. Right now I'm thinking of possible operations in terms of the directory structure of the FTP server whereas end points need to be based on what operations can be performed on tables.